components/director/internal/domain/application/service_test.go

```diff
@@ -441,14 +437,7 @@ func TestService_Update(t *testing.T) {
 	testCases := []struct {
 		Name               string
 		AppRepoFn          func() *automock.ApplicationRepository
-		WebhookRepoFn      func() *automock.WebhookRepository
-		APIRepoFn          func() *automock.APIRepository
-		EventAPIRepoFn     func() *automock.EventAPIRepository
-		DocumentRepoFn     func() *automock.DocumentRepository
-		LabelRepoFn        func() *automock.LabelRepository
-		FetchRequestRepoFn func() *automock.FetchRequestRepository
-		LabelServiceFn     func() *automock.LabelUpsertService
-		Input              model.ApplicationInput
+		Input              model.ApplicationUpdateInput
```

Comment:
This test doesn't check if we really updated the application properties.
I would remove `mock.MatchedBy` func